### ISSUE TITLE
Stream Deck 6.0

### ISSUE THREAD

**Issue by ghost** [ChimeraVT/streamdeck-nanoleaf#9]
Since Stream Deck update 6.0, nanoleafs are no longer found, not even manually.

**Comment by Baptiewright** [comment]
Yes this is a known issue with v6 and is being worked on by the StreamDeck team.  It affects all plugins that use XML/http requests.    Hopefully there will be a fix soon, also I'll be working on a new version supporting the new StreamDeck+ features.

**Comment by Fangh** [comment]
Any news on this ? we are on 6.8.1
